kubernetes/client/clientset/clientset.go

```diff
@@ -1,96 +1,23 @@
 package clientset
 
 import (
-	composev1beta1 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta1"
-	composev1beta2 "github.com/docker/cli/kubernetes/client/clientset/typed/compose/v1beta2"
-	"github.com/golang/glog"
-	"k8s.io/client-go/discovery"
-	"k8s.io/client-go/rest"
-	"k8s.io/client-go/util/flowcontrol"
+	api "github.com/docker/compose-on-kubernetes/api/client/clientset"
 )
 
 // Interface defines the methods a compose kube client should have
 // FIXME(vdemeester) is it required ?
```

Comment:
These `FIXME`'s can be removed, as they'll have to be done in the new location now